src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,31 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import ladder.domain.generator.LineGenerator;
+
+public class Ladder {
+
+    private final List<Line> lines;
+
+    public Ladder(final LineGenerator lineGenerator, final Players players, final Height height) {
+        this.lines = generateLines(lineGenerator, players, height);
+    }
+
+    private List<Line> generateLines(final LineGenerator lineGenerator, final Players players, final Height height) {
+        final List<Line> lines = new ArrayList<>();
+        final int numberOfPlayers = players.numberOfPlayers();
+
+        for (int i = 0; i < height.getValue(); i++) {
+            lines.add(lineGenerator.generate(numberOfPlayers));
+        }
```

Comment:
LineGenerator.generate에 넘기는 numberOfPlayers를 generate 메서드 내부에서는 directionCount로 사용하고 있어요.

```suggestion
        int directionCount = numberOfPlayers - 1;
        for (int i = 0; i < height.getValue(); i++) {
            lines.add(lineGenerator.generate(directionCount));
        }
```
위와 같은 방식으로 구현하면 LineGenerator를 사용할때 혼란스럽지 않으면서 코드도 깔끔해질 것 같은데 어떠세요?